test/unit-tests.ts

```diff
@@ -0,0 +1,23 @@
+import * as assert from "assert";
+import * as functions from "./../src/functions/functions";
+import * as mocha from "mocha";
+
+describe("Test Excel Custom Functions", function () {
+    describe("Test add function", function () {
+        it(`Add function should return expected value`, async function () {
+            assert.equal(functions.add(5, 6), 11);
+        });
+    });
+    describe("Test logMessage function", function () {
+        it(`LogMessage function should return expected value`, async function () {
```

Comment:
Consider: "basic message".  Other test cases might be "empty string", "contains Emoji character", etc.